### PR TITLE
Add new gifs to random_gif.rb

### PR DESCRIPTION
**Pull request by wuz**
## What type of PR is this?
- [x] Feature

## Description

This adds a few more success/celebration gifs to the random gif file!

## Related Tickets & Documents
N/A

## Added to documentation?

- [x] no documentation needed

## What gif best describes this PR or how it makes you feel?

Here's all the ones I added. They all are a good representation!

### CODE REVIEW

File: app/labor/random_gif.rb [hunk 1]
```diff
@@ -13,6 +13,12 @@ def initialize(_action = "congratulations")
       "y8Mz1yj13s3kI" => { aspect_ratio: 0.750 },
       "111ebonMs90YLu" => { aspect_ratio: 0.750 },
       "Sk5uipPXyBjfW" => { aspect_ratio: 0.422 },
+      "l0K48FkLfeSCzRA4M" => { aspect_ratio: 0.573 },
+      "3o7qDRd1DlF7P2TP3O" => { aspect_ratio: 0.517 },
+      "26h0qt6UOumsbJkyI" => { aspect_ratio: 0.442 },
+      "3o7qDOzZAhOslfIgMM" => { aspect_ratio: 0.807 },
```
Comment (Glennmen) [inline app/labor/random_gif.rb]: I think this `aspect_ratio` should be 0.829.

Width: 480
Height: 398

Comment (Zhao-Andy) [reply app/labor/random_gif.rb]: I agree with this. BTW @Glennmen did you know about GitHub's suggestion feature? 

screen shot 2019-03-04 at 11 34 07 am

Comment (Glennmen) [reply app/labor/random_gif.rb]: @Zhao-Andy I did read about it in the past but was looking for it and couldn't find it (and was lazy and didn't look it up)... My bad!

Comment (Glennmen) [inline app/labor/random_gif.rb]:
```suggestion
      "3o7qDOzZAhOslfIgMM" => { aspect_ratio: 0.829 },
```

Comment (Zhao-Andy) [reply app/labor/random_gif.rb]: All good, happy to share it :) I'm a big fan of it!

Comment (benhalpern) [reply app/labor/random_gif.rb]: Hey @wuz can you commit this change?

Comment (wuz) [reply app/labor/random_gif.rb]: Done! Thanks @Glennmen! Thanks for the reminder @benhalpern!

**Pull request merged**